bin/run_indexing.py

```diff
@@ -221,78 +224,49 @@ def run_find_cuboids(bosslet_config, args):
                                             channel_params) 
         print('lookup_key is: {}'.format(lookup_key))
 
-    find_cuboid_args = get_find_cuboid_args(bosslet_config,
-                                            lookup_key)
-    #print(find_cuboid_args[1])
+    start_args = get_start_args(bosslet_config, lookup_key)
+    #print(start_args[1])
 
-    print('Starting Index.FindCuboids . . .')
+    print('Starting Index.Start . . .')
     sfn = bosslet_config.session.client('stepfunctions')
     resp = sfn.start_execution(
-        stateMachineArn=find_cuboid_args[0],
-        input=find_cuboid_args[1]
+        stateMachineArn=start_args[0],
+        input=start_args[1]
     )
     print(resp)
 
 
 def resume_indexing(bosslet_config):
     """
-    Resume indexing a channel or channels.  If the CuboidsKeys queue is not
-    empty, indexing will resume on those cuboids identified in that queue.
+    Rework required.  Probably want to reconnect cuboids keys queue and the
+    ids queues to their respective lambdas to resume.
 
     Args:
         bosslet_config (BossConfiguration): Bosslet configuration object
     """
-    resume_args = get_common_args(bosslet_config)
-    resume_args['queue_empty'] = False
-    arn = resume_args['id_supervisor_step_fcn']
-
-    print('Resuming indexing (starting Index.Supervisor) . . .')
-    sfn = bosslet_config.session.client('stepfunctions')
-    resp = sfn.start_execution(
-        stateMachineArn=arn,
-        input=json.dumps(resume_args)
-    )
-    print(resp)
+    #resume_args = get_common_args(bosslet_config)
+    raise NotImplementedError()
 
 
 def stop_indexing(bosslet_config):
     """
-    Stop the indexing process, gracefully.  Index.CuboidSupervisors will not
-    be stopped, so the entire index process will not terminate, immediately.
-    Only the Index.Supervisor and any running Index.DequeueCuboid step 
-    functions will be halted.  This allows the indexing process to be resumed.
+    This needs to be reworked.  Once the cuboid keys queue is populated, the
```

Comment:
Good question. If there's no other channels being indexed, the queues could be purged. Otherwise, there's not really a good way to kill it without affecting other channels being indexed.